docker-compose.yml

```diff
@@ -30,20 +31,20 @@ services:
       - ./frontend:/app/
     env_file:
       - .env
+    environment:
+      - PORT=8080
+      - BACKEND_URL=http://backend:3000
     ports:
-      - "8080:8080"
-    links:
-      - bookclub-voting-backend
+      - 8080:8080
 
-  bookclub-voting-postgres:
-    container_name: postgres
+  postgres:
     image: postgres:13-alpine
     env_file:
       - .env
     ports:
-      - "5432:5432"
+      - 5432:5432
     volumes:
-      - pgdata:/var/lib/postgresql/data
+      - ./here/:/var/lib/postgresql/data
```

Comment:
Any particular reason to not use the named volume?

Reply:
Yes, for debugging.
Oops.
Removed